### ISSUE TITLE
Incorrect free fall damage

### ISSUE THREAD

**Issue by vvs-** [opentomb/OpenTomb#88]
Something is wrong in the free fall damage calculations. In *Atlantean Stronghold* move to `setEntityPos(player,46158,49343,14127,270,0,0)` and then climb down. Try to shimmy right as far as possible and then let go. The damage should be exactly half of maximum HP, but it's not. Also, it's actually impossible to climb down from the rightmost position, that's why the instructions are so complicated.

**Comment by Lwmte** [comment]
Yes, falling damage is purely subjective, I didn't do any comparisons with original code. Must fix it some day!

**Comment by stohrendorf** [comment]
Decompiled TR1 calculation: https://github.com/stohrendorf/EdisonEngine/blob/master/src/abstractstatehandler.cpp#L3307
